libs/sdk/skyux-eslint/project.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "skyux-eslint",
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "libs/sdk/skyux-eslint/src",
+  "projectType": "library",
+  "tags": ["sdk", "npm"],
+  "targets": {
+    "build": {
+      "executor": "@nx/js:tsc",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "dist/libs/sdk/skyux-eslint",
+        "tsConfig": "libs/sdk/skyux-eslint/tsconfig.lib.json",
+        "packageJson": "libs/sdk/skyux-eslint/package.json",
+        "main": "libs/sdk/skyux-eslint/src/index.ts",
+        "assets": ["libs/sdk/skyux-eslint/*.md"]
```

Comment:
This should also include `collection.json`.

Reply:
Good call.